data/config-schema/src/main/java/net/corda/schema/configuration/MessagingConfig.java

```diff
@@ -95,14 +95,14 @@ private StateManager() {
         public static final String JDBC_PASS = DB_PROPERTIES + ".pass";
 
         public static final String JDBC_URL = DB_PROPERTIES + ".jdbc.url";
-        public static final String JDBC_DRIVER = DB_PROPERTIES + "jdbc.driver";
+        public static final String JDBC_DRIVER = DB_PROPERTIES + ".jdbc.driver";
         public static final String JDBC_DRIVER_DIRECTORY = DB_PROPERTIES + ".jdbc.directory";
         public static final String JDBC_PERSISTENCE_UNIT_NAME = DB_PROPERTIES + ".jdbc.persistenceUnitName";
         public static final String JDBC_POOL_MAX_SIZE = DB_PROPERTIES + ".pool.maxSize";
         public static final String JDBC_POOL_MIN_SIZE = DB_PROPERTIES + ".pool.minSize";
         public static final String JDBC_POOL_IDLE_TIMEOUT_SECONDS = DB_PROPERTIES + ".pool.idleTimeoutSeconds";
         public static final String JDBC_POOL_MAX_LIFETIME_SECONDS = DB_PROPERTIES + ".pool.maxLifetimeSeconds";
-        public static final String JDBC_POOL_KEEP_ALIVE_TIME_SECONDS = DB_PROPERTIES + ".pool.keepAliveTimeSeconds";
+        public static final String JDBC_POOL_KEEP_ALIVE_TIME_SECONDS = DB_PROPERTIES + ".pool.keepaliveTimeSeconds";
```

Comment:
I noticed we've introduced slightly inconsistent naming for this property, I just want to keep it consistent

Reply:
I'm not sure we can actually change the others (crypto and rbac) at this stage, that's probably a breaking change?

Reply:
It's unfortunately spread around the codebase quite a bit, I'd prefer we named it proper camelcase but my gut tells me to keep it consistent. I will change it back since we can't really change the others, at least stateManager will be correct.